tests/improver-blend-adjacent-points/01-help.bats

```diff
@@ -51,15 +52,18 @@ positional arguments:
   COORDINATE_TO_BLEND_OVER
                         The coordinate over which the blending will be
                         applied.
+  CENTRAL_POINT         Central point at which the output from the triangular
```

Comment:
Should you say this in in parameter units too, and relates to the coordinate to blend over?

Reply:
Corrected.